docs/interpreter/bigquery.md

```diff
@@ -44,6 +44,21 @@ Zeppelin is built against BigQuery API version v2-rev265-1.21.0 - [API Javadocs]
 
 In a notebook, to enable the **BigQuery** interpreter, click the **Gear** icon and select **bigquery**.
 
+### Setup service account credentials
+
+In order to run BigQuery interpreter outside of Google Cloud Engine you need to provide authentification crednetials,
+by [following this instructions](https://developers.google.com/identity/protocols/application-default-credentials):
+
+ - Go to the [API Console Credentials page](https://console.developers.google.com/project/_/apis/credentials)
+ - From the project drop-down, select your project.
+ - On the `Credentials` page, select the `Create credentials` drop-down, then select `Service account key`.
+ - From the Service account drop-down, select an existing service account or create a new one.
+ - For `Key type`, select the `JSON` key option, then select `Create`. The file automatically downloads to your computer.
+ - Put the *.json file you just downloaded in a directory of your choosing. This directory must be private (you can't let anyone get access to this), but accessible to your Zeppelin instance.
```

Comment:
maybe you need to change 
"*.json" -> "`*.json`" 
or it will show the whole sentence as italic letter.